controllers/progresses.js

```
@@ -267,4 +269,51 @@ const getProgressBydDateController = async (req, res) => {
+ * @param {Object} req.body - The request body containing an array of progress records.
+ * @param {Array<ProgressRequestBody>} req.body.records - Array of progress records to create.
+ * @param {Object} res - The HTTP response object.
+ * @returns {Promise<void>} A Promise that resolves when the response is sent.
+ */
+const createBulkProgress = async (req, res) => {
+  if (req.userData.roles.archived) {
+    return res.boom.forbidden(UNAUTHORIZED_WRITE);
+  }
+
+  const { records } = req.body;
+  
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)